### PR TITLE
[typescript] avoid error in future Typescript 4.1

### PR DESCRIPTION
**Pull request by Manu1400**
see https://github.com/microsoft/TypeScript/issues/41155

### CODE REVIEW

**Comment by Manu1400** [comment]
In Typescript@next (4.1.0-beta) : https://www.typescriptlang.org/play?ts=4.1.0-beta#code/MYewdgzgLgBGMF44EMBuBLA5sqIBOAUAQKYAeADvrACbEBmyArgDax2NjBTrgybFQAMsjCZGyfhAAUASgBcMaHnSiA2gF0NMAN4EYMUJFh5iEFlAVKVmDVqQai+w9BjMRYicURwAdG9Hi-DAAPsG+jBDEeMIBniFhYD4ARnggAO6R0e6BXqG+EACe0MQAtjEe-How6HQwUv4VXgCECEgctHQqxNQyOlX6JmasPuQRABZSqg056jJVAL6OvtOeEEs1dQCCeHjIBT7oENu7BVKJK5Iyvbr6+gD0dzAAylDIeNyiMHSpJTAAwmMfuhGL8ABwANgANDAoGMol5DjBwMwCjBkIprMwvKhkMxGAj4AADMBoLA4fB+bKeQnVeAASU4IEwYHQuBgJRAtDk-RgDxhcJgOLxXhAtWJpOwuDwlNi-EJPhgABU4SY6PgvLDsbj8dUINVmerqLT+YiUulMj4eRspE0pIcAUCQbIYAAyF26h0gErAkoQgDyeAAcsQ0lFZFc+rdboNzCNxmcZY09cg9VY1LMeYt9PMYMRmJFqrUpFACuRiKLllTJIhWjAAORpzB16485ywC6mbznKudlOKKDKURWosdiAtqMDUyx0YQCajnwQcjMVlSOuQ5tzKNZmCLKomKCMPDwGOsAjzIA

File: src/sources/languages.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function getLanguages(): string[][] {
     // Starting from Chromium 86, there is only a single value in `navigator.language` in Incognito mode:
     // the value of `navigator.language`. Therefore the value is ignored in this browser.
     if (!(isChromium() && isChromium86OrNewer())) {
-      result.push(n.languages)
+      result.push(n.languages as string[])
```
Comment (Finesse) [inline src/sources/languages.ts:17]: Thanks for letting us know! But ignoring TypeScript warnings isn’t a good solution. Please make a writable copy of the array instead:

```js
result.push([...n.languages])
```

Comment (Finesse) [inline src/sources/languages.ts:17]: Thanks for letting us know! But ignoring TypeScript warnings isn’t a good solution. Please make a writable copy of the array instead:

```js
result.push([...n.languages])
```

**Review by Finesse — CHANGES_REQUESTED** [review]

**Review by Toronto00 — APPROVED** [review]

**Comment by Valve** [comment]
Closing due to inactivity. Please feel free to reopen with additional updates and/or comments.

**Pull request closed without merging**